runtime/src/startup/mod.rs

```diff
@@ -55,6 +57,10 @@ macro_rules! stack_size {
     }
 }
 
+pub fn handle_main_return<T: Termination>(result: T) -> ! {
+    Termination::complete::<TockSyscalls>(result)
+}
```

Comment:
Add a comment to `handle_main_return` mentioning that although it is `pub`, it is actually an implementation detail of `libtock_runtime` used by `set_main!`.

Reply:
Done.